src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search__bar-big Big search bar data-qa_big"
```

Comment:
These classes are useless.

```suggestion
      class="search__bar-big"
```